scripts/main/album.js

```diff
@@ -1286,7 +1286,6 @@ album.updatePhoto = function (data) {
 	if (album.json) {
 		$.each(album.json.photos, function () {
 			if (this.id === data.id) {
-				this.filesize = data.filesize;
```

Comment:
Shouldn't you update `filesize` in `deepCopySizeVariant` above?

Reply:
Oh, nice catch, sorry for this.